providers/cncf/kubernetes/tests/unit/cncf/kubernetes/operators/test_pod.py

```diff
@@ -55,9 +55,9 @@
 from airflow.utils.types import DagRunType
 
 from tests_common.test_utils import db
-from tests_common.test_utils.version_compat import AIRFLOW_V_3_0_PLUS
+from tests_common.test_utils.version_compat import AIRFLOW_V_3_0_PLUS, AIRFLOW_V_3_1_PLUS
 
-if AIRFLOW_V_3_0_PLUS:
+if AIRFLOW_V_3_0_PLUS or AIRFLOW_V_3_1_PLUS:
```

Comment:
```suggestion
if AIRFLOW_V_3_0_PLUS:
```

`AIRFLOW_V_3_0_PLUS`  is defined as, in `AIRFLOW_V_3_1_PLUS` case, the `AIRFLOW_V_3_0_PLUS` will also be `True`.
```python
AIRFLOW_V_3_0_PLUS: bool = get_base_airflow_version_tuple() >= (3, 0, 0)
```

Reply:
@jason810496 Good point. The compatibility tests were failing earlier since I hadn’t added this part `AIRFLOW_V_3_1_PLUS `
Related slack discussion : [here](https://apache-airflow.slack.com/archives/C06K9Q5G2UA/p1754830025810419?thread_ts=1754827049.450009&cid=C06K9Q5G2UA)

I’d love to hear your thoughts on this.

Reply:
~If I don't get it wrong, we only need add compatible if/else for `XComModel.get_many` usage, and the rest of the setup will still be the same for both `AIRFLOW_V_3_0_PLUS` and `AIRFLOW_V_3_1_PLUS` .~

Sorry, I got the point, the `AIRFLOW_V_3_1_PLUS` usage here is good.